src/styles/modules/header.scss

```diff
@@ -0,0 +1,66 @@
+@import "../utils/variables";
+
+.header {
+  background: $white;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav {
```

Comment:
The '.nav' class could be nested inside the '.header' class to follow the BEM methodology more accurately.